packages/lib-utils/src/app/api-discovery/index.ts

```diff
@@ -187,7 +193,7 @@ const startAPIDiscovery = () => (dispatch: DispatchWithThunk) => {
     .catch((err) => consoleLogger.error('API discovery startAPIDiscovery polling failed:', err));
 };
 
-export const initAPIDiscovery: InitAPIDiscovery = (storeInstance) => {
+export const initAPIDiscovery: InitAPIDiscovery = (storeInstance, preferenceList = []) => {
   consoleLogger.info(`API discovery waiting ${API_DISCOVERY_INIT_DELAY} ms before initializing`);
   const initDelay = new Promise((resolve) => {
     window.setTimeout(resolve, API_DISCOVERY_INIT_DELAY);
```

Comment:
@karelhala can we reduce this initial timeout? 15 seconds is too long.

Reply:
Sure thing!